packages/devtools_app/lib/src/inspector/flutter/inspector_data_models.dart

```diff
@@ -102,16 +103,23 @@ class LayoutProperties {
             : node?.childrenNow
                 ?.map((child) =>
                     LayoutProperties(child, copyLevel: copyLevel - 1))
-                ?.toList(growable: false);
+                ?.toList(growable: false) {
+    for (var child in children) {
+      child.parent = this;
+    }
+  }
 
+  LayoutProperties parent;
```

Comment:
This doesn't appear to be used. Can you remove it?

Reply:
Good catch, yes I can remove this